beacon-light-client/plonky2/circuits_executables/bin/final_layer.rs

```diff
@@ -102,6 +106,9 @@ async fn async_main() -> Result<()> {
 
     let epoch = BigUint::div(final_input_data.slot, 32u32).to_u64().unwrap();
     let commitment_proof: ValidatorProof = fetch_proof(&mut con, 1, epoch).await?;
+
+    proof_storage.set_protocol("".to_string());
+
     let commitment_proof_bytes = proof_storage
         .get_proof(commitment_proof.proof_index)
         .await?;
```

Comment:
The protocol prefix needs to be in the `proof_index`. The commitment mapper proofs will have no protocol prefix, thus emptying the prefix won't be necessary since it's not part of the proof_storage api. I also recommend renaming `proof_index` to `proof_key` instead since it makes a little more sense to me.